static/js/camera.js

```diff
@@ -21,7 +24,7 @@ function updateSnapshotLoop() {
       let snapshot = document.getElementById("snapshot-image")
       snapshot.src = "/camera/snapshot?"+ new Date().getTime();
       snapshotCount++;
-      if (snapshotCount < 200) {
+      if (snapshotCount < 200 || !timeout) {
```

Comment:
This is a little confusing - could we rename the variable to "timeoutDisabled" to improve clarity of intent?